src/ethereum_test_tools/spec/blockchain_test.py

```diff
@@ -155,6 +155,8 @@ def make_block(
                 txs=to_json(txs),
                 env=to_json(env),
                 fork=fork,
+                block_number=Number(env.number),
+                block_timestamp=Number(env.timestamp),
```

Comment:
Point 2. How about doing this instead? Or do you think we'll need the Fork object in evaluate at some point in the future (instead of just the name)?
```suggestion
                fork_name=fork.fork(block_number=Number(env.number), block_timestamp=Number(env.timestamp))
```

Reply:
Yes, I think I was saving it to open up for the possibility of some transition tool implementing a different name at some point, but I don't think this will ever happen now that we have them all supported and we can just raise a PR on the specific tool to align its naming of the fork.